langlib/lang.annotations/src/main/ballerina/annotations.bal

```diff
@@ -99,6 +99,8 @@ public const annotation record {
     string label;
     # icon path relative to module's resource directory
     string iconPath?;
+    # kind of the data
+    "text"|"password"|"file" kind?;
```

Comment:
From original issue,


> 
> The default would be `"text"` when there is no `@display` annotation for the given field.
> 

@hasithaa @indikasampath2000 @KavinduZoysa Does this mean, the default value `text` will be handled from UI side? or should it be a default value of the `display` record itself?

Reply:
We can handle this from UI as well. But I think better to have a default value to `text` as you have mentioned.

```
 "text"|"password"|"file" kind = "text";
```

@hevayo what do you think?

Reply:
No, we shouldn't have a default value here. For example, It doesn't make sense when a @display annotation on a service declaration with `kind = "text"`.